### DISCUSSION TITLE
I can't get site_config(s) to show in the GUI

### DISCUSSION THREAD

**Discussion by JanHitako** [K-vanc/Tempest-EPG-Generator#119] [Q&A]
I set up the dockerized image within a docker-compose because I need to run some other dockerfiles.

For volume, I tried both this:
"./tempest_config/:/var/www/html/tempest_config/" (relative path
and this "home/ubuntu/apps/myTV/tempest_config/:/var/www/html/tempest_config/" (absolute path)

Which is the path to my actual tempest_config folder which is in my project directory.
which is like this:
myTV/
---docker-compose.yml
---[some bunch of other things]
---tempest_config/
------site_config/
---------many site config files I downloaded here (xml and php)

But, when I access the tempest.php, then go to EPG Generator, the list is always empty, I'm clueless as to why.

Anyone can help?

**Comment by K-vanc** [comment]
Hi,

It seemed to me as you have some permission issue. For mounting a folder, second part is fixed and it will be always;

`:/var/www/html/tempest_config/`

on your example for absolute path, i see as you didn't start with "/" which is mandatory for all UNIX distros so it should be;

`/home/ubuntu/apps/myTV/tempest_config/:/var/www/html/tempest_config/`

Now, above line will mount your "/myTV/tempest_config/" folder into Tempest's "/tempest_config/" folder. On your first call of Tempest(via browser or cli), Tempest will create its folder structure and you will see generated subfolders inside your "/myTV/tempest_config/" directory. Now you can copy/paste siteconfig php and xml files into auto-generated "/site_config/" subfolder. If you directly copy/pasted into your main folder, then they are in wrong place so Tempest will not see them. Basicly, do not copy/paste any siteconfig before auto-folder creation. As example you can check below .md;

[Siteconfigs/0_How to use](https://github.com/K-vanc/Tempest-EPG-Generator/tree/dcee183ef9e86caa8a4a23343c1b189bea193483/Siteconfigs/0_How%20to%20use)

If you cannot see created subfolders inside your mount folder, you can check below issue for OpenMediaVault permissions, You didnt mention your system so i am just wild-guessing as usual; 

https://github.com/K-vanc/Tempest-EPG-Generator/issues/107#issue-2241915283

**Comment by K-vanc** [comment]
Hi, 

I didn't hear from you so considering as issue is resolved.
